xla/service/gpu/nccl_api_stub.cc

```diff
@@ -1,11 +1,8 @@
 /* Copyright 2024 The OpenXLA Authors.
-
```

Comment:
I guess removing the blank lines here is not intended?

Reply:
Sure :)  Done